crates/primitives/commitments/Cargo.toml

```diff
@@ -17,6 +17,7 @@ mp-felt = { workspace = true }
 mp-hashers = { workspace = true }
 mp-transactions = { workspace = true }
 starknet-crypto = { workspace = true }
+starknet-ff = { workspace = true }
```

Comment:
Why is it not optional anymore?
The changes in `crates/primitives/commitments/src/lib.rs` don't use it. Do they?

Reply:
Corrected it